### PR TITLE
Remove spec.helm.valuesFiles field and update spec.helm.period field

### PR DESCRIPTION
**Pull request by xinnywinne**
description

### CODE REVIEW

File: pkg/api/configsync/v1alpha1/helmconfig.go [hunk 1]
```diff
@@ -44,11 +44,6 @@ type Helm struct {
 	// +optional
 	Values *unstructured.Unstructured `json:"values,omitempty"`
 
-	// valuesFiles is a list of URL to Helm value files.
-	// Values files must be in the same repository with the Helm chart or in a public repository.
-	// +optional
-	ValuesFiles []string `json:"valuesFiles,omitempty"`
```
Comment (mikebz) [inline pkg/api/configsync/v1alpha1/helmconfig.go:50]: if this is removed, how do I provide values?

**Review by nan-yu — APPROVED** [review]
/lgtm

File: pkg/api/configsync/v1alpha1/helmconfig.go (hunk 1 above)
Comment (xinnywinne) [reply pkg/api/configsync/v1alpha1/helmconfig.go:50]: Here are some discussions relate to this field https://github.com/GoogleContainerTools/kpt-config-sync/pull/22#discussion_r946253655. User will provide values through "values" field

**Review by tiffanny29631 — APPROVED** [review]
/lgtm

**Pull request merged**